{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "941fb32b_4e1e64b8",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 34
      },
      "lineNbr": 0,
      "author": {
        "id": 27900
      },
      "writtenOn": "2024-07-30T17:22:13Z",
      "side": 1,
      "message": "recheck pypi network issues",
      "revId": "c5b12b841cc670af088a7491f0b857e1d527fd7d",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}